searchlib/src/tests/features/euclidean_distance/euclidean_distance_test.cpp

```diff
@@ -43,7 +43,7 @@ struct SetupFixture
 
 TEST_F("require that blueprint can be created from factory", SetupFixture)
 {
-    EXPECT_TRUE(FTA::assertCreateInstance(f.blueprint, "euclideanDistance"));
+    EXPECT_TRUE(FTA::assertCreateInstance(f.blueprint, "squaredEuclideanDistance"));
```

Comment:
this seems unintended

Reply:
Yes, clion was not smart enough, or too smart for me.